### DISCUSSION TITLE
What happened to EPUB files UI?

### DISCUSSION THREAD

**Discussion by ghost** [sumatrapdfreader/sumatrapdf#2712] [Q&A]
After recent update I opened an EPUB file in Sumatra but it opened like a PDF would, the yellow background and full page text UI wasn't there. How do I turn it on again? Or is it something I did? Or is it gone for good?
I see one similar Discussion to this(https://github.com/sumatrapdfreader/sumatrapdf/discussions/2461#discussion-3991150)but it was marked "Unanswered" and also they were talking about facing this problem in older version but I am facing it in the latest version.
I have tried upto five older version in Portable format and they all are opening Epub files in PDF view.

Please Help!

**Comment by kjk** [comment]
Switched to a new epub rendering engine which is more capable and supports text selection and search.

But didn't have the time to update the old UI to support this new rendering engine. Maybe later.

**Reply by zwxr** [reply]
Yeah, prefer old UI for EPUB files. it would be nice if you can switch between old UI and new UI in the incoming update.

**Reply by okapi-kakapo** [reply]
Same, I recently updated to the latest version to find EBOOK UI gone. After tinkering with advanced settings and finding no way to mimic the old facing view for ebooks, I reverted to an older version.
I love that KjK added the text selection and copy features for ebooks but Id like to request that you bring back the ebookUI if possible.
Thank you for all your work, absolutely love Sumatra.

**Reply by gatotj** [reply]
I'm new to SumatraPDF (thanks KjK), in my quest to find epub reader close to Apple's iBook experience on iPad, specifically on page separation.

I found that this new engine (in v3.4 above) suits this purpose well, compared to v.3.3.3 (with it's customizable setting). I think, I will stick with the default value, even when customization will be available again.

FYI. Using 3:2 screen-ratio laptop.

Thanks again KjK!

**Comment by aberfan** [comment]
Is there any time line for a fix for this? Sumatrapdf was amazing; best epub reader, period.  Now, every book renders in just a thin vertical line on my widescreen monitor, with no way of 'fixing' it. You can't 'fit width' or the text is obnoxiously large.

**Comment by Nojarosann** [comment]
Hey the person from the other problem here. I actually resolved my other problem by changing the fileinto a PDF and then into .EPUB again in some random site i found.

Tried the same here and the problem is not going away. it seems to really be a problem with the current version. i'll try to download the versioni had back then and update here if the problem's fixed

Edit: Fixed.
[SumatraPDF-3.3.3-64.zip](https://github.com/sumatrapdfreader/sumatrapdf/files/10430364/SumatraPDF-3.3.3-64.zip)
